src/model/dfcir/lib/dfcir/conversions/FIRRTLStubsGeneratorPass.cpp

```diff
@@ -0,0 +1,164 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the Utopia HLS Project, under the Apache License v2.0
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021-2024 ISP RAS (http://www.ispras.ru)
```

Comment:
2021-2024 -> 2024

Reply:
Fixed here and in `templates/stubs.tpl`.